### ISSUE TITLE
code for youtubevos training and evaluation

### ISSUE THREAD

**Issue by pixelsmaker** [microsoft/transductive-vos.pytorch#1]
Do you have any plan to release the code for youtubevos training and evalutation?

**Comment by eastOffice** [comment]
Hello,

We have released the training & evaluation code for Youtube-VOS under youtube-vos branch. Before running evaluation on validation set, you need to run the dataset/clean_ytb.py to remove images before the first annotation.
